packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -161,110 +211,140 @@ body {
         }
       }
     }
+    &:focus-within {
+      border-color: var(--aa-primary-color);
+      box-shadow: var(--aa-primary-color-alpha) 0 0 0 2px,
+        inset var(--aa-primary-color-alpha) 0 0 0 2px;
+      outline: currentColor none medium;
+    }
   }
 }
 
 // ----------------
-// Panel
+// 4. Panel
 // ----------------
-
 .aa-Panel {
-  position: absolute;
-  // reset
-  * {
-    box-sizing: border-box;
-    margin: 0;
-    padding: 0;
-  }
-  button {
-    appearance: none;
-    background: none;
-    border: 0;
-  }
+  @extend %init;
 
-  &::after {
-    background-image: linear-gradient(
-      var(--aa-background-color-alpha-0),
-      var(--aa-background-color)
-    );
-    border-radius: 3px;
-    bottom: 0;
-    content: '';
-    height: 0.75em;
-    pointer-events: none;
-    position: absolute;
-    width: 100%;
-    z-index: 99;
+  background-color: var(--aa-background-color);
+  border-radius: calc(var(--aa-spacing) / 4);
+  box-shadow: var(--aa-panel-shadow);
+  margin: 8px 0 0;
+  overflow: hidden;
+  position: absolute;
+  transition: opacity 200ms ease-in, filter 200ms ease-in;
+  @media screen and (prefers-reduced-motion) {
+    transition: none;
   }
-  .aa-PanelLayout {
-    background-color: var(--aa-background-color);
-    border-radius: 3px;
-    box-shadow: var(--aa-panel-shadow);
+  @at-root .aa-PanelLayout {
     height: 100%;
-    margin-top: var(--aa-spacing-half);
+    margin: 0;
     max-height: var(--aa-panel-max-height);
-    padding-bottom: var(--aa-spacing-half);
-    padding-top: var(--aa-spacing-half);
+    overflow-y: auto;
+    padding: 0;
+    padding: var(--aa-spacing-half) var(--aa-spacing) var(--aa-spacing)
+      var(--aa-spacing-half);
     position: relative;
     text-align: left;
 
-    .aa-PanelLayoutResults {
-      overflow-x: hidden;
-      overflow-y: auto;
-      width: 50%;
+    &.aa-PanelLayout-cols-two--golden {
```

Comment:
From what I've noticed so far, the naming convention is namespaced BEM-ish: `prefix-PascalCaseElement--camelCaseModifier`. It works well for a library, but this class name here (and the next one for column styling) breaks out of the model. It should probably be something like `aa-PanelLayoutColumns--two`, `aa-PanelLayoutColumns--twoGolden`, or `aa-PanelColumns--two`, `aa-PanelColumns--twoGolden` (dropping the `Layout` for column classes).

Since stylelint is installed, you can [enforce the class name](https://github.com/algolia/autocomplete/blob/next/.stylelintrc.json#L16) with a stricter regular expression. [This one](https://regex101.com/r/8z8zBJ/1) should cover it.

Reply:
Thanks for the Style Lint configuration.

See b8728ca30373012c0eb1b8dc032371eeefe1961e and 17cdd2106b6f0542602d11ac4222306ff341ea16.